examples/pi/intel-entrypoint.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set_intel_vars=/opt/intel/oneapi/setvars.sh
+if [ -f $set_intel_vars ]; then
+  source $set_intel_vars
+fi
+
+function resolve_host() {
```

Comment:
I think this still can cause issues sometimes, i.e. sometimes there is a window when the launcher is able to resolve it's own hostname, but workers are not. This happens to me usually in the second run if I schedule two runs in a row.

It should be fine though, since this is only an example

Reply:
Are you sure that was the problem?
When I only had the check for the launcher, I was getting flaky startups. Now that I have checks for every workers as well, the job starts every time. I couldn't really debug what was going on with just the check for the launcher, as Hydra  doesn't log the output of the ssh calls :(